components/codeflare/codeflare.go

```diff
@@ -43,7 +44,15 @@ func (d *CodeFlare) ReconcileComponent(owner metav1.Object, client client.Client
 	if enabled {
 		// check if the CodeFlare operator is installed
 		// codeflare operator not installed
-		found, err := deploy.SubscriptionExists(client, CodeflareOperatorNamespace, CodeflareOperator)
+		dependentOperator := CodeflareOperator
+		platform, err := deploy.GetPlatform(client)
+		if err != nil {
+			return err
+		}
+		if platform != deploy.OpenDataHub && platform == "" {
```

Comment:
Perhaps it is my very limited knowledge of Go, but wouldn't the second part of the condition suffice? I mean `platform == ""` implies that `platform != deploy.OpenDataHub`, unless `deploy.OpenDataHub == ""`?

Or was it meant as `platform != deploy.OpenDataHub || platform == ""`?

Pardon my Go ignorance :)

Reply:
thanks for pointing this out!

Reply:
Thank you, LGTM now! :)